ministryofjustice/analytics-platform-control-panel-frontend
# Fix restart in docker-compose

This makes using docker-compose for development less annoying

## What

* Mount host directories in container, so files can be monitored for changes
* Run with `NODE_RESTART=1` in docker-compose, to restart on changes to files
* Move `nodemon` config into `package.json` to avoid change to Dockerfile

## How to review

1. Run with `docker-compose up` (you may need to `docker-compose build` first)
2. Change a file in `app/` or `lib/`
3. See the node app restart
